post-scan-actions/aws-python-slack-notification/handler.py

```diff
@@ -0,0 +1,45 @@
+import urllib3
+import json
+import os
+http = urllib3.PoolManager()
+from botocore.exceptions import ClientError
+
+def lambda_handler(event, context):
+
+    url = os.environ['SLACK_URL']
+    channel = os.environ['SLACK_CHANNEL']
+    username = os.environ['SLACK_USERNAME']
+    
+    for record in event['Records']:
+
+        #Message details from SNS event
+        message = json.loads(record['Sns']['Message'])
+        findings = message['scanning_result'].get('Findings')
+
+        #ARN info to get AWS Account ID
+        arn = json.dumps(record['EventSubscriptionArn'])
+        account_id = arn.split(":")[4].strip()
+
+        if findings:
+        
+            for finding in message['scanning_result']['Findings']:
+                body_text = '''\
+                WARNING\n
+                AWS Account ID: {account}\n
+                File URL: {file_url}\n
+                Malware Name: {malware}\n
+                Malware Type: {type}\
+                '''.format(account=str(account_id),
```

Comment:
This gives some wired indent to the slack message. Try this
```python
import textwrap
# ...
                body_text = textwrap.dedent('''
                WARNING
                AWS Account ID: {account}
                File URL: {file_url}
                Malware Name: {malware}
                Malware Type: {type}
                ''').format(account=str(account_id),
```

Reply:
Just text here and works for me. Thank you for the tip